troposphere/static/js/components/login/LoginScreen.jsx

```diff
@@ -65,7 +65,7 @@ export default React.createClass({
     },
     attemptOpenstackLogin: function(username, password, projectName, provider, onLoginError) {
```

Comment:
We should really clean the dead code since we are not using the projectName and never will. That could be a domain name in the future. But we shouldn't really keep it in the master branch imo.